tools/build_api.py

```diff
@@ -411,26 +411,25 @@ def get_mbed_official_release(version):
               RELEASE_VERSIONS
     """
 
+    # we ignore version for Mbed 6 as all targets in targets.json file are being supported
+    # if someone passes 2, we return empty tuple, if 5, we keep the behavior the same as 
+    # release version is deprecated and all targets are being supported that are present
+    # in targets.json file
+
+    if version == '2':
+        return tuple(tuple([]))
+
     mbed_official_release = (
         tuple(
             tuple(
                 [
                     TARGET_MAP[target].name,
-                    tuple(transform_release_toolchains(
-                        TARGET_MAP[target], version))
+                    tuple(['ARM', 'GCC_ARM'])
```

Comment:
just wonder why do we use hard-coded tuple here?  instead of calling the function? and IAR been dropped?
We can fix the function if need?

Reply:
I wanted to avoid refactoring at the moment (so changes just to one function to get the functionality needed). `transform_release_toolchains` does not have much meaning anymore as only ARM or GCC ARM is supported, so this can be hardcoded here